### ISSUE TITLE
bug in image function

### ISSUE THREAD

**Issue by bwlewis** [Paradigm4/SciDBR#34]

**Comment by bwlewis** [comment]
More details. Image doesn't work with arrays that contain '*' bounds.

**Comment by bwlewis** [comment]
Here is an example of this bug:

```r
 x = build("(random() % 1000)/1000.0", c(1000,1000), type="double", chunksize=c(250,250))
 a = image(unbound(x))

Error in scidbquery(query, afl, async = FALSE, save = "lcsv+", release = 0,  : 
  UserQueryException in file: src/query/parser/Translator.cpp function: matchOperatorParam line: 1049
Error id: scidb::SCIDB_SE_QPROC::SCIDB_LE_WRONG_OPERATOR_ARGUMENT2
Error description: Query processor error. Parameter must be constant with type 'int64'.
save(regrid(project(redimension(build(<val:double>[i=0:999,250,0,j=0:999,250,0],(random() % 1000)/1000.0),<val:double>[i=0:*,250,0,j=0:*,250,0]),val),Inf,Inf,sum(val)),'/tmp//shim_output_buf_aP4vz7',0,'lcsv+')

```

**Comment by bwlewis** [comment]
fixed in commit  a2082ef
